com.unity.shadergraph/Editor/Generation/Processors/GenerationUtils.cs

```diff
@@ -102,7 +102,7 @@ internal static void GenerateShaderStruct(StructDescriptor shaderStruct, ActiveF
 
                         //check for a semantic, build string if valid
                         string semantic = subscript.HasSemantic() ? $" : {subscript.semantic}" : string.Empty;
-                        structBuilder.AppendLine($"{subscript.type} {subscript.name}{semantic};");
+                        structBuilder.AppendLine($"{subscript.interpolation} {subscript.type} {subscript.name}{semantic};");
```

Comment:
Note for me: tracking change compare to this PR: https://github.com/Unity-Technologies/Graphics/pull/3467/files#diff-13f5ff3fe703f5b644e6dd0281cd393c6987650f8cefa0d1f4e596332e95ebb6R105